### PR TITLE
Phase1 Digis 9_2_0

### PR DESCRIPTION
**Pull request by fioriNTU**
equivalent of #18949 for Online integration

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -73,6 +73,7 @@
 # Real data raw to digi
 process.load("EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi")
 process.siPixelDigis.IncludeErrors = True
+process.siPixelDigis.UsePhase1 = True
```
Comment (davidlange6) [inline DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py]: Hi @fioriNTU - would it not work just to change

process = cms.Process("PIXELDQMLIVE")

to 
from Configuration.StandardSequences.Eras import eras
process = cms.Process("PIXELDQMLIVE", eras.Run2_2017")

as to get all the 2017 configuration changes without having to repeat them in your cfgs?

**Comment by davidlange6** [comment]
-1 
pending answers to above comments.

File: DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py (hunk 1 above)
Comment (fioriNTU) [inline DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py]: sorry @davidlange6  I did not see this comment before, my bad. 
The era does not work out of the box, I get the following error:

python encountered the error: <type 'exceptions.AttributeError'>
'Process' object has no attribute 'gtEvmDigis'

I'll try to fix it in the coming days

Comment (davidlange6) [inline DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py]: hi @fioriNTU - it seems to run ok for me (not convinced I processed events but my job run over a bunch of files from run 295613 eating cpu..)

**Comment by fioriNTU** [comment]
@davidlange6 the "normal client" seems to be ok with eras, I have some problem with the Tracking client in which gtEvmDigis seems to be needed (probably for some L1 tech bit selection?) I am testing the client without those digis, if everything is ok I will submit a new PR later and close this one. Of course using the era mechanism would be more reliable for our clients

**Comment by fioriNTU** [comment]
@davidlange6 I have submitted a new PR #19134 that superseed this one which I will close now

**Pull request closed without merging**